### PR TITLE
Fix equip via load out builder

### PR DESCRIPTION
**Pull request by kyleshay**
Per https://github.com/DestinyItemManager/DIM/issues/854#issuecomment-250558932 equip to current character was not working as expected. This attempts to fix it. Don't have time yet to test.

### CODE REVIEW

File: app/scripts/minmax/dimMinMax.controller.js [hunk 1]
```diff
@@ -314,7 +314,7 @@
       },
       equipItems: function(set) {
         ngDialog.closeAll();
-        var loadout = { items: {} };
+        var loadout = { items: {}, name: 'Automatic Loadout Builder' };
```
Comment (bhollis) [inline app/scripts/minmax/dimMinMax.controller.js]: Probably want to use translate service here.

File: app/scripts/minmax/dimMinMax.controller.js [hunk 2]
```diff
@@ -325,14 +325,15 @@
...
-              items: items
-            }, true);
-          });
+        loadout = angular.copy(loadout);
+
+        _.each(loadout.items, function(val) {
+          val[0].equipped = true;
+        });
+
+        return $q.when(dimStoreService.getActiveStore()).then(function(current) {
```
Comment (bhollis) [inline app/scripts/minmax/dimMinMax.controller.js]: `getActiveStore` doesn't return a promise, it just returns a value, so all this promise business is unnecessary.

Comment (kyleshay) [inline app/scripts/minmax/dimMinMax.controller.js]: just c&p'd what was there, did look into that though, and was a bit confused. can update.

File: app/scripts/minmax/dimMinMax.controller.js [hunk 3]
```diff
@@ -314,7 +314,7 @@
       },
       equipItems: function(set) {
         ngDialog.closeAll();
-        var loadout = { items: {} };
+        var loadout = { items: {}, name: $translate.instant('LoadoutAppliedAuto') };
```
Comment (bhollis) [inline app/scripts/minmax/dimMinMax.controller.js]: Oh cool, I didn't know about `instant`.

**Pull request merged**